src/Framework/Features.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Build.Framework
+{
+    /// <summary>
+    /// The status of a feature.
+    /// </summary>
+    public enum FeatureStatus
+    {
+        /// <summary>
+        /// The feature is not found.
+        /// </summary>
+        NotFound,
```

Comment:
What is the intended use case for having separate values for `NotFound` and `NotAvailable`?

Reply:
The difference between using `NotAvailable` and `NotFound` statuses is that `NotAvailable` indicates that the feature is definitely not available, whereas `NotFound` indicates that the feature was not found in our list of features. We may not add all the features to the dictionary, so `NotFound` does not necessarily mean that the feature is not available.

Reply:
It is slightly confusing. What about `Undefined`?

Reply:
Yes, that is a better name for it. Thanks!

Reply:
Renamed it to `Undefined`.